editor/inspector/assets/material.js

```diff
@@ -56,6 +70,25 @@ exports.methods = {
         this.dirtyData.uuid = '';
     },
 
+    async updateCustomInspector() {
+        this.$.customPanel.hidden = false;
+        this.$.section.hidden = true;
+        this.$.materialDump.hidden = true;
+        try {
+            const relatePath = normalize((await this.getCustomInspector()).replace('packages://', ''));
+            const name = relatePath.split(sep)[0];
+            const packagePath = Editor.Package.getPackages({ name, enable: true })[0].path;
+            const path = join(packagePath, relatePath.split(name)[1]);
```

Comment:
before join path, need to judge the package is exist or not.

Reply:
There is no need. A try catch is used